### PR TITLE
Remove (possibly) unused files?

### PR DESCRIPTION
**Pull request by akleeman**
grep seems to indicate that these files aren't used.

Is anyone still using them? or know how to use them?

@swift-nav/estimation

### CODE REVIEW

**Comment by benjamin0** [comment]
^ @fnoble @mookerji

File: gnss_analysis/utils.py [hunk 1]
```diff
@@ -46,8 +46,8 @@ def sphere_b_covariance(var=0.0025):
 
 
 def dd_phi_cov(n, var):
```
Comment (mookerji) [inline gnss_analysis/utils.py]: This is not used.

File: gnss_analysis/utils.py [hunk 2]
```diff
@@ -46,8 +46,8 @@ def sphere_b_covariance(var=0.0025):
...
-    one_row = np.array([[1.0]*n])
-    return (np.eye(n) + one_row.T.dot(one_row))*var
+    one_row = np.array([[1.0] * n])
+    return (np.eye(n) + one_row.T.dot(one_row)) * var
 
 
 def get_N_from_b(phase, de, b, b_cov=None, phi_var=None):
```
Comment (mookerji) [inline gnss_analysis/utils.py]: Nor this

File: gnss_analysis/utils.py [hunk 3]
```diff
@@ -58,24 +58,24 @@ def get_N_from_b(phase, de, b, b_cov=None, phi_var=None):
...
 def normalize(x):
-    return x/np.sqrt(x.dot(x))
+    return x / np.sqrt(x.dot(x))
 
 
-l2pi = np.log(2*np.pi)
+l2pi = np.log(2 * np.pi)
 
 
 def neg_log_likelihood(x, sigma):
```
Comment (mookerji) [inline gnss_analysis/utils.py]: Nor this

File: gnss_analysis/utils.py [hunk 4]
```diff
@@ -96,7 +96,7 @@ def neg_log_likelihood(x, sigma):
 
 
 def get_de(ref_ecef, alm, sats_w_ref_first, time):
```
Comment (mookerji) [inline gnss_analysis/utils.py]: Nor this

**Comment by mookerji** [comment]
Indeed! A bunch of it is unused. Feel free to purge it.

**Pull request merged**